Ginger/GingerCoreNET/ActionsLib/ActScreenShot.cs

```diff
@@ -115,7 +115,17 @@ public override void PostExecute()
 
             if (!Directory.Exists(DirectoryPath))
             {
-                Directory.CreateDirectory(DirectoryPath);
+                try
+                {
+                    Directory.CreateDirectory(DirectoryPath);
+                }
+                catch (Exception ex)
+                {
+                    this.Status = Amdocs.Ginger.CoreNET.Execution.eRunStatus.Failed;
+                    Error = "Invalid Folder Path.";
+                    throw ex;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider using 'throw;' to preserve the stack trace.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5233273)